src/app/launcher/cancel-overlay/cancel-overlay.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="cancel-overlay" *ngIf="launcherComponent.showCancelOverlay === true">
+<div class="cancel-overlay" *ngIf="launcherComponent.showCancelOverlay  ">
```

Comment:
Extra white space has been added to all places where you remove ```=== true```. Can you please remove the white space.

Reply:
Thanks @vikram-raj for paying attention.  Fixed in 7abcddb197f05d0c3f3157bcf674cb4468c74a58.

I am wondering - do you know if there are any tools which we can employ to our UI projects so that we save ourselves from noise like that?

Ideally, we could have:

  * automation to format the code as a hook
  * bot which can find some more serious linter issues and comment directly in the review

Reply:
No, I don't know any tool for the template which can handle this. But we can handle these things in our ```tslint``` for ```.ts``` file.